src/utils/displayIdentity.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styled from "styled-components";
+
+import { Identity } from "@lib/@types";
+import { RightChevron } from "@src/components/display/fewlines/Icons/RightChevron/RightChevron";
+
+export const displayIdentity = (identity: Identity): JSX.Element => {
+  return (
+    <>
+      <IdentityValue primary={identity.primary} status={identity.status}>
+        {identity.value}
+      </IdentityValue>
+      <RightChevron />
+    </>
+  );
+};
+
+const IdentityValue = styled.p<Pick<Identity, "primary" | "status">>`
+  ${(props) =>
+    props.primary &&
+    `
+      font-weight: ${props.theme.fontWeights.semibold};
+    `}
+
+  ${(props) =>
+    props.status === "unvalidated" &&
+    `
+      color: ${props.theme.colors.lightGrey};
+    `};
+`;
```

Comment:
I'm a bit confused about the location of this file. Since it returns a React JSX Component, why don't you put it in `src/component/display/` folder instead of `src/utils/` ? :)

Reply:
OK this version would be more suitable dc70ff9
The commit is loaded but let me break down my reasoning:
- I created 2 new display components `<StandardIdentitiesSection />` and `<SocialIdentitiesSection />` (I'll create their stories right after), therefore, I could get rid of `displayIdentity()` and `displaySocialIdentities()` from `@src/utils`
- Those aforementioned components will apply a specific display whether for social identities or standard ones, thanks to this logic, I could have skimmed `IDENTITIES_SECTION_CONTENT` from 3 extras entries per identity type
- `<IdentitiesSection />` is now rendering either `<StandardIdentitiesSection />` or `<SocialIdentitiesSection />` according to the identity type given in props
- For more readability I created a new `type IdentitiesSectionContent` that I use in all those 3 component (not `<StandardIdentitiesSection />` yet though, I forgot, my bad, I'll fix it in an upcoming commit

Reply:
@Fuzznimp you can check this out as well, this commit provides solutions for your changes request

Reply:
Yes, I will do it :). I'm almost done with improving the GraphQL's errors object returned, which is going to changes a couple of things in here (mostly `sortIdentities`).